.travis.yml

```diff
@@ -2,51 +2,51 @@ language: node_js
-  # VISUAL_DIFF_S3_SECRET
-  - secure: hETcU1ZRPPz7ipofSeUj8fLkdM9JJda/Qg/oXU83FqZ/y/LZya6jMJjBfcHgEGsCLmVOPLw4lj2+pcyA6ZprJuujdS56elnbWPYUa4iwCbPN7ayfFQQjQx3jo3LYd7jFlCilgMT+Q+s+SnVfmV1Wa91N5WOpINpi7NzIarBRhWVeoiCQJmaabAX2pIn4G5VnBIE562SJULq+05Ucavl9zWrSFHDsMqMtcVcHG7TI6BExENJf76iutEk/YqWdaSvk5LY4xLrO29tThZBVJ7lX5wQGa3MkmlJkHZqxoONBWbGsKfrwh5I7QHExkj8PYYfvpYHQd+VugFrSkDTW3wHsoe/cmrzVe8aEbpHfUSPpO5HOkTdOqy92NVjyh4MOE7hJexmQsYRD+zjFzL4AS876hOdjrf/FS5ume843MvZseSZiCpK7uHTQWwVmitib/dNJdy8YyWoAyceWHpz46QOArrECVYOFhMKYuPrNy3gYY05kCLHCEW24pVN1ToRW2Gp6Z3Z3YblfWShjFHl3eU8riOPeCU9RaBV1k1UuP0C5POyFDPS12K8glPNIJ6dbp6GyI6KKW3iYuX/O3G4w/evO/AMy9nQPL4tUOPU5gQsPvr4Z77+bluP9VqEd3o1VBC6IBwLEcc8tzO+5EAeRbj9THU7zDmoIQH0oRHpIo+elg/8=
-  # GITHUB_RELEASE_TOKEN (2374......46ed)
-  - secure: Q0Nl6PYZhAvS2rUprp7yxAnFmaaWlRJ7Vek1y8uQ0h2Rm4hraRDTwmcFSKOdOgHnI75Wzn5qCcTDHnQMVu1tHUf7/Z1Fnk/2cG5Y5G/R877jihoaaSNdhlbBPV92EZ5+tSc51H3hmChoK4FePKv45KQLsz59txOYmsVzKd8ryKTXzrDSH5IA3ssyBLfRTT6RbRDdTRBOpoTR4zPBkWCj9vB1Pd4g01jZPk2oP1bd8sIfJp1vrEzRCOrR7FiuDIBNArKTqOoLt3rlNry4uZpwrZmmA5b0L7xNM1C1PUxMjCKZV7mT5MthVx9R2NVOrddxAE0P7z/6PK1V86oJYwoSySOedLM4AYCwddpZNzqIR5aEAM33rvbRYuyCURt/1w59SXb5zftigdjbVg7wbvxb7sxWOEbhg+LHlno0+8U1r2WU1OY3BZBvsEKCV4jGGJpSyhvQpJ0gyWfsNk3Kt37OFgG0ZBNdXgIfQ+jSzan8puoR8wUbz4DyuYOV/z5Y4LdswFR3q4/eo1FjjzBBz+fXDodXxUm2XT/lxz5uL23y5EUqfT6vIBXd2XhlQ/izS0PCckzPILSz+nOL/I4Hq/q2Zfzu+9kD/sL6G3n1c28LbQnNOsuXyfoBozVZvwPc8wXWf4jIJlWGUXCoyG2zRJkN5D64GnSDvzu6+TThOYU8ZNQ=
-deploy:
-  provider: npm
-  email: d2ltravisdeploy@d2l.com
-  skip_cleanup: true
-  api_key:
-    # 57c4......d343
-    secure: IitMXMxHrtsjCy3eucLs/mxja8e27DRpxT1YnA7HM6nCpKEr+SzPaHqV4tSNfX8fmKvqhyR3SRijPWnGahaZNVAPlN0l01MSEaG419o3QIKFzSgpNZLjNe5BsGTH3CFC5yPgtrU6ILW2Et1pb1+oixghXSifybxxan4aJKwBuri1DnrijmBnlEyog1Wglra1e7GzwZnEohDitHWKqO2V6uE+v1LmDtnHMzKoEjEeD/iHk1/oKdfsbZLEcZ+l6xU3gORBMskG+XT8EPgPOpfnuCiL0tqJ+ZFcP2hbNrzMMKs/k5TAmvL0DJyvmDfjlBXLik4aHKX6GmYGfhfktBCEN4hl8rmo9eBoQxJuzWWnCJq5BxVRGxaBZg7yZK5qZYR8cUwS5t241LqDgMK4rQP6J6cImCZ1Fe93t9yMd3GsurSHIByHwwQ3tuY4Xj3f6Nr/skGYrS9azqRtREWsRY4rsiudcKuTQGsDfLMpEN6ZGZQBXrYgKZpCrgIUiK5PyggbzTfOfpSrxe4oBD6IIb6bITYPPU5rXnE4EMOLY6AUn5OBxR/ACkooccFa7gHxlgVaNAy+RRIFk4sW6NAbADZXzy1U+jp8QsCzZBU0gmjNLShD0uuUfWSdv75qq3eKlwBzrlI6TpADoGmsFxoO3Yc6d0YGOmcxWVu6ZRuqeyVTZ9I=
-  on:
-    repo: BrightspaceUI/core
-    tags: true
+    - stage: tests
```

Comment:
In the past I've thought about changing this so we have... "unit-tests" and "visual-difference-tests".  🤷‍♂

Reply:
But if you end up moving it into a job in the same stage, then "tests" makes a lot of sense.  Didn't notice the rename before.

Reply:
Haha well technically this stage is "lint" and "unit tests" but yeah, I'll look at the naming if I can get them into the same stage. It's going to require a change to the bot.